site/src/InspectorPanel.css

```diff
@@ -61,4 +61,7 @@
 
   box-shadow: 0 2px 4px black;
   color: white;
+
+  scrollbar-color: var(--ifm-color-secondary-darkest)
```

Comment:
Only substantive change. This lies within the .theme-dark .inspector-panel property definition. Other changes are formatting